R/R-ParallelLogger/Portfile

```diff
@@ -3,20 +3,17 @@
 PortSystem          1.0
 PortGroup           R 1.0
 
-R.setup             github OHDSI ParallelLogger 165b3ebdb18352bf7295eef75fd2f13b84d046b4
-version             3.3.0
-revision            0
+R.setup             github OHDSI ParallelLogger 3.3.0 v
```

Comment:
the "version" doesn't change here but the "distfile" and "checksums" do because you change the download  location. Thus this should be treated as a "stealth update". But ideally you don't do this at all...

Reply:
@reneeotten Got it, I will add a stealth update hack. Thought revbumping is enough.